icu4c/source/i18n/unicode/msgfmt.h

```diff
@@ -243,6 +243,20 @@ class NumberFormat;
  * </table>
  * <p>
  *
+ * <h4>Argument formatting</h4>
+ *
+ * <p>Arguments are formatted according to their type, using the <code>toString()</code>
```

Comment:
There is no toString() in C++. Please rephrase to what works here.
Maybe something like "... using default formatters for those types, unless otherwise specified."  ?